### PR TITLE
[CXF-8758]:Add wiremock-standalone test dependency to fix more systests

### PR DESCRIPTION
**Pull request by jimma**

### CODE REVIEW

**Comment by jimma** [comment]
@reta  This is the failure I was looking at today and I'll continue to have a look tomorrow. I created another JIRA issue for this :https://issues.apache.org/jira/browse/CXF-8781, and I'll try to fix with another PR.

**Comment by reta** [comment]
> @reta This is the failure I was looking at today and I'll continue to have a look tomorrow. I created another JIRA issue for this :https://issues.apache.org/jira/browse/CXF-8781, and I'll try to fix with another PR.

Thanks @jimma, I think we shouldn't merge this one till we have a solution (having failing tests is fine for now), may be to share some thoughts on that. I doubt that just adding `wiremock-standalone` will help us here (unless it shades the Jetty dependencies). May be what we need is to downgrade jetty to 9.x for this particular set of of modules (there is no server-side dependencies on the CFX here).

**Comment by jimma** [comment]
@reta It looks we have 2 failures left : https://ci-builds.apache.org/job/CXF/job/CXF-JDK17-PR/111. Do you see any dependency missing or we need to implement something about jsonb we missed for MP restclient ?

**Comment by jimma** [comment]
@reta  And another thing  I commented out is the` testng.xml` along with `RetryAnalyzer`.  Using this configuration is problematic for tck tests and gets about 26 failures. Do you see any issue if we remove this testng.xml for tck tests?

**Comment by reta** [comment]
> @reta It looks we have 2 failures left : https://ci-builds.apache.org/job/CXF/job/CXF-JDK17-PR/111. Do you see any dependency missing or we need to implement something about jsonb we missed for MP restclient ?

This is awesome, I will take a look shortly

File: systests/microprofile/client/weld/pom.xml [hunk 1]
```diff
@@ -85,17 +92,65 @@
             <artifactId>johnzon-jsonb</artifactId>
```
Comment (reta) [inline systests/microprofile/client/weld/pom.xml:92]: Could you please change this one to:

```
<dependency>
            <groupId>org.apache.johnzon</groupId>
            <artifactId>johnzon-jsonb</artifactId>
            <classifier>jakarta</classifier>
            <exclusions>
                <exclusion>
                    <groupId>*</groupId>
                    <artifactId>*</artifactId>
                </exclusion>
            </exclusions>
        </dependency>
        <dependency>
            <groupId>org.apache.johnzon</groupId>
            <artifactId>johnzon-mapper</artifactId>
            <classifier>jakarta</classifier>
        </dependency>
```

(the same as other POM), thanks

**Comment by reta** [comment]
> > @reta It looks we have 2 failures left : https://ci-builds.apache.org/job/CXF/job/CXF-JDK17-PR/111. Do you see any dependency missing or we need to implement something about jsonb we missed for MP restclient ?
> 
> This is awesome, I will take a look shortly

Should be fixed by https://github.com/apache/cxf/pull/1020

File: systests/microprofile/client/weld/pom.xml (hunk 1 above)
Comment (jimma) [reply systests/microprofile/client/weld/pom.xml:92]: Done.

**Comment by jimma** [comment]
> Should be fixed by https://github.com/apache/cxf/pull/1020

@reta  This is awesome! Thanks for the quick fix.

**Review by reta — APPROVED** [review]

**Comment by reta** [comment]
@jimma just merged https://github.com/apache/cxf/pull/1020, could you rebase please? thanks!

**Comment by jimma** [comment]
@reta  Rebased. Thanks.

**Comment by reta** [comment]
Green :)

**Comment by jimma** [comment]
> Green :)

This is great !  I get this PR merged.

**Pull request merged**